src/js/components/Select/SelectContainer.js

```diff
@@ -36,22 +40,58 @@ const SelectOption = styled(Button)`
   width: 100%;
 `;
 
-const ClearButton = ({ clear, onClear, name, theme, setFocus }) => {
+const ClearButtonContainer = styled(Box)``;
+const ClearButtonText = styled(Text)``;
+const ClearButtonComponent = ({
```

Comment:
Our naming convention doesn't add the suffix "Component" to components. Can this just be "ClearButton"?